client/src/components/publications/form/CategoryPage.js

```diff
@@ -88,15 +109,26 @@ const CategoryPage = ({ next, prev, data, closeModal }) => {
             <Form.Group>
               <Form.Label>
                 {' '}
-                {Object.keys(categoryType).find(
-                  (category) => category.toUpperCase() === values.category.type
-                )}{' '}
+                {Object.keys(categoryType)
+                  .filter(
+                    (category) =>
+                      category.toUpperCase() === values.category.type
+                  )
+                  .map((category, idx) => {
+                    return `${
+                      category.charAt(0) + category.slice(1).toLowerCase()
+                    } title`;
+                  })}{' '}
               </Form.Label>
               <Form.Control
+                id="placeholder-text"
                 type="text"
                 name="category.categoryTitle"
-                placeholder={Object.keys(categoryType).find(
-                  (category) => category.toUpperCase() === values.category.type
+                placeholder={renderPlaceholderText(
+                  Object.keys(categoryType).find(
+                    (category) =>
+                      category.toUpperCase() === values.category.type
+                  )
```

Comment:
Would it be possible to use a plain old JS object to define the mapping of the placeholder text? The `renderPlaceholderText` doesn't seem to be as readable as just an object IMO.

Reply:
Can do!